### PR TITLE
MDAnalysis.analysis.rms.rmsd docs fixed version entries

### LINKED ISSUE #4014: MDAnalysis.analysis.rms.rmsd docs do not show versionchanged entries
## Expected behavior ##

Docstring shows any versionchanged entries that have been added to the docstring.

## Actual behavior ##

Despite having `versionchanged` entries, the docs for https://docs.mdanalysis.org/dev/documentation_pages/analysis/rms.html#MDAnalysis.analysis.rms.rmsd don't seem to be showing them.

It's possibly down to the number of empty lines there should be between the doctest block and the versionchange entries, but it will need further investigation.

## Current version of MDAnalysis ##

Happening in the current dev branch of MDAnalysis.

### PR DESCRIPTION
**Pull request by MeetB7**
Fixes  #4014 

Changes made in this Pull Request:
 - syntax error in version entries

PR Checklist
------------
 - [x] Tests?
 - [x] Docs?
 - [x] CHANGELOG updated?
 - [x] Issue raised/referenced?

### CODE REVIEW

**Review by orbeckst — CHANGES_REQUESTED** [review]
Hi @MeetB7 , this looks good, nice and clean work! 👍 

Please also do the following
- [x] add yourself to the AUTHORS file
- [ ] add your GitHub handle to the top of CHANGELOG (where you see the other people who contributed to the 2.5.0 release); you don't need an entry but you should be credited as a contributor
- [x] introduce yourself on the mdanalysis-developer list, please (reference this PR in your message), see https://www.mdanalysis.org/#participating

Please ping me with @orbeckst when you've done everything. This will notify me, otherwise I might not remember to look at the PR (sorry, I am busy with many different things)

Once you've done all of the above, I can merge your first contribution!

**Comment by orbeckst** [comment]
Hi @MeetB7 , please also add yourself to CHANGELOG and **Resolve the conflicts**. Otherwise it cannot be merged.

**Comment by MeetB7** [comment]
Hey @orbeckst i have made the changes and my pr is ready to merge.
Thank you!! 
Ps: sorry for delay

File: package/CHANGELOG [hunk 1]
```diff
@@ -13,7 +13,7 @@ The rules for this file:
   * release numbers follow "Semantic Versioning" http://semver.org
 
 ------------------------------------------------------------------------------
-??/??/?? IAlibay, pgbarletta, mglagolev, hmacdope, manuel.nuno.melo, chrispfae, ooprathamm
+??/??/?? IAlibay, pgbarletta, mglagolev, hmacdope, manuel.nuno.melo, chrispfae, ooprathamm, MeetB7
```
Comment (orbeckst) [inline package/CHANGELOG]: Keep it in 79 columns
```suggestion
??/??/?? IAlibay, pgbarletta, mglagolev, hmacdope, manuel.nuno.melo, chrispfae, 
         ooprathamm, MeetB7
```

**Review by orbeckst — CHANGES_REQUESTED** [review]
- [x] Did you check the built docs and visually confirmed that the version changed now show up? There's a read the docs check that leads you to the built docs. Please paste the link to the page where the rmsd docs are displayed and confirm that it looks good.

- [x] There's one line break issue in the CHANGELOG. I created a suggestion for a fix. Just commit my suggestion by pressing the nice  "Commit suggestion" button.

Looks good to me, thank you, I am just waiting for you to do the two things.

**Comment by orbeckst** [comment]
To find the docs, identify the **docs/readthedocs.org:mdanalysis** check and click on *Details*.

On the subsequent page, find the tiny gray link *View Docs* (see image below).
rtd_pr

**Comment by MeetB7** [comment]
Yeah the version entries now show up in the docs @orbeckst 
https://mdanalysis--4022.org.readthedocs.build/en/4022/documentation_pages/analysis/rms.html#functions

**Review by orbeckst — APPROVED** [review]

**Comment by orbeckst** [comment]
Thank you for your first contribution @MeetB7 , well done 🎉

**Comment by MeetB7** [comment]
> Thank you for your first contribution @MeetB7 , well done 🎉

Thank you for guiding me through it. I can't wait to contribute more in future.

**Pull request merged**